src/messages/serenity.proto

```diff
@@ -49,8 +49,6 @@ message Contention{
     NETWORK = 4;
   }
 
-  optional WorkID victim = 1;
```

Comment:
I deleted too much. Signal observer may not be entity that is responsible for detecting aggressor (qos strategy is) - but for sure it can detect victim :)